after/plugin/dracula.vim

```diff
@@ -15,15 +23,19 @@ if exists('g:loaded_fzf') && ! exists('g:fzf_colors')
         \ 'spinner': ['fg', 'Label'],
         \ 'header':  ['fg', 'Comment'] }
 
+  let s:laststatus = &laststatus
+  let s:showmode = &showmode == 0 ? 'showmode' : 'noshowmode'
+  let s:ruler = &ruler == 0 ? 'ruler' : 'noruler'
```

Comment:
Seems like quite a bit of overhead for such a teeny tiny edge case.. How often during the course of usual activity is laststatus hotswapped? What would even be the reason for ever doing that?

Reply:
I'd rather remove this completely than add that into the codebase.

Reply:
`'laststatus'` is just an example; if we're going to futz with global options we should be saving and restoring them, regardless of a user's reasons for changing options on the fly.

However, I agree, it might be more worth it just to remove this. Thinking about it, I'm not sure why we were messing with options for `fzf` anyways.

Reply:
Ah, you meant more or less make a function that could take care of _all_ global settings.. Hm, that might come in handy. What do you think?

Yeah, I added the FZF thing because I use it personally and adding that little lesser-known snippet makes the output look a lot nicer. 

PS: Sorry for being sort of snappy earlier. Long week! :white_flag:

Reply:
If we need to change global settings, it would be handy. If we decide we don't need to (I think FZF is the only case where we currently are changing them) then YAGNI.

I'd say let's nix the FZF autocmds that don't have much to do with Dracula (people can always add them to a vimrc or whatever if they want them).

RE: 🏳️ no problem! We all have those 😄 I definitely saw it more as a technical discussion then anything else; I can be both long-winded and very terse when I try to make a point, which doesn't always come across well. Hope your weekend is more relaxed!